sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3106,6 +3106,27 @@ pi_result piDeviceGetInfo(pi_device Device, pi_device_info ParamName,
                   ZeDevicePciProperties.address.function);
     return ReturnValue(AddressBuffer);
   }
+
+  case PI_EXT_INTEL_DEVICE_INFO_FREE_MEMORY: {
+    if (getenv("ZES_ENABLE_SYSMAN") == nullptr) {
+      zePrint("Set ZES_ENABLE_SYSMAN=1 to obtain free memory.\n");
+      return PI_ERROR_INVALID_VALUE;
```

Comment:
Should this be a backend specific error instead, so the user gets notified rather than it looking like a backend crash?

Reply:
I can do that, so that the message is shown as a part of SYCL trace, not just with ZE_DEBUG=1 currently. Although it is rather a warning, not an error. Basically, in case the ZES_ENABLE_SYSMAN is not set to 1 the has_aspect, which users must check, would return false.

Reply:
Added it as a plugin-specific message